src/sockets/commands/helperFunctions.ts

```diff
@@ -0,0 +1,23 @@
+import { Command } from './types';
+
+export function adjustCommandPrefix(
+  command: Command,
+  oldPrefix: string,
+  newPrefix: string,
+): Command {
+  // Throw error if invalid command is passed
+  if (!command.command.startsWith(oldPrefix)) {
+    throw new Error(`Incorrect command prefix. Expected: ${oldPrefix}`);
+  }
+
+  if (!command.help.startsWith(`/${oldPrefix}`)) {
+    throw new Error(`Incorrect command help prefix. Expected: /${oldPrefix}`);
+  }
+
+  const shallowCopy = { ...command };
```

Comment:
Can you construct a new object so that we specifically only "copy" the run function? 
This allows us to specify exactly what we want to copy, rather than blanket copying the entire object.

Reply:
I instead created a new Command object with a reference pointing to the other Command's run function